internal/translation/audit/audit.go

```diff
@@ -24,15 +19,6 @@ type AuditLog struct {
 	auditAPI admin.AuditingApi
 }
 
-// NewAuditLogService creates an AuditLog from credentials and the atlas provider
```

Comment:
If we do this everywhere we might be able to remove the code at the `translation` parent package.

Maybe we should pin a reminder for that in a Jira. But it does not affect this PR.